lib/stytch/rbac_local.rb

```diff
@@ -54,17 +54,17 @@ def perform_authorization_check(
       raise Stytch::PermissionError, authorization_check
     end
 
-    # Performs an authorization check against the project's policy and a set of roles. If the
+    # Performs an authorization check against the project's policy and a set of scopes. If the
     # check succeeds, this method will return. If the check fails, a PermissionError
-    # will be raised. It's also possible for a TenancyError to be raised if the
-    # subject_org_id does not match the authZ request organization_id.
-    # authorization_check is an object with keys 'action', 'resource_id', and 'organization_id'
+    # will be raised. This is used for OAuth-style consumer authorization.
+    # authorization_check is an object with keys 'action' and 'resource_id'
```

Comment:
Comment looks wrong - this check uses roles and doesn't do OAuth-style authorization